tests/server.rs

```diff
@@ -15,10 +15,11 @@ use std::time::Duration;
 use futures_channel::oneshot;
 use futures_core::ready;
 use futures_core::future::BoxFuture;
-use futures_util::future::{self, Either, FutureExt};
-use futures_util::stream::StreamExt;
-use futures_util::try_future::{self, TryFutureExt};
-//use futures_util::try_stream::TryStreamExt;
+use futures_util::future::{self, Either, FutureExt, TryFutureExt};
+#[cfg(feature = "unstable-stream")]
+use futures_util::stream::StreamExt as _;
+// TODO: remove once tokio is updated to futures 0.3
```

Comment:
It looks like we are just waiting for a new `0.2.0-alpha.7`. https://github.com/tokio-rs/tokio/issues/1708

Reply:
Yes, that'd be ideal. I've been meaning to try against a git hash pin, but I haven't gotten around to it yet.

Importing both works but isn't very pretty.